packages/gatsby-plugin-graphql-codegen/src/graphql-codegen.config.test.js

```diff
@@ -69,26 +68,11 @@ it('takes in options and returns a function that runs codegen for the schema', a
 
   await generateFromSchema(mockSchema)
 
-  expect(fs.ensureDir).toMatchInlineSnapshot(`
-    [MockFunction] {
-      "calls": Array [
-        Array [
-          "example-directory",
-        ],
-      ],
-      "results": Array [
-        Object {
-          "type": "return",
-          "value": undefined,
-        },
-      ],
-    }
-  `)
   expect(fs.writeFile).toMatchInlineSnapshot(`
 [MockFunction] {
   "calls": Array [
     Array [
-      "example-directory/example-types.ts",
+      undefined,
```

Comment:
i'm very curious on why this changed to `undefined`. this seems unexpected. can you confirm this is correct?

Reply:
You're right this isn't right, I'll run the test again